cgmanifest.json

```diff
@@ -3200,6 +3380,266 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "libX11",
```

Comment:
Aren't most `libX*` packages in the CBL-MarinerCoreUI repo and not here? Also, I think these aren't part of the toolchain, right? The CBL-MarinerCoreUI repo has got its own `cgmanifest.json` btw.

Reply:
Yes, I think a number of these libX* packages are also in CoreUI. However, they are downloaded as part of the toolchain build process in Core, and are required to compile OpenJDK from source.